src/BoostInterface.hpp

```diff
@@ -0,0 +1,92 @@
+//Copyright (c) 2020 Ultimaker B.V.
+#ifndef BOOST_INTERFACE_HPP
+#define BOOST_INTERFACE_HPP
+
+#include <boost/polygon/voronoi.hpp>
+#include <boost/polygon/polygon.hpp>
+
+#include "utils/IntPoint.h"
+#include "utils/PolygonsSegmentIndex.h"
+#include "utils/polygon.h"
+
+
+using CPoint = arachne::Point;
```

Comment:
Is an `arachne::Point` different from a `cura::Point`? I'm unable to find this definition (and so is my compiler).

Reply:
I don't think it _is_ different. It was probably needed when libArachne was going to be it's own thing --and thus needed to copy small parts of CuraEngine-- rather than a part of the engine.

Reply:
This `CPoint` is not used anywhere so I've removed it.
Now that it properly compiles (thanks), my compiler claims that `arachne::Point` is an alias of `cura::Point`. I've removed these definitions then.